File: backend/src/main/java/ca/metricalsky/winston/repository/fetch/FetchActionRepository.java
package ca.metricalsky.winston.repository.fetch;

import ca.metricalsky.winston.entity.fetch.FetchActionEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface FetchActionRepository extends JpaRepository<FetchActionEntity, Long> {

}

File: database/src/main/kotlin/ca/metricalsky/winston/database/repository/fetch/FetchActionRepository.kt
package ca.metricalsky.winston.database.repository.fetch

import ca.metricalsky.winston.database.entity.fetch.FetchActionEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface FetchActionRepository : JpaRepository<FetchActionEntity, Long>
